server/rpc_test.go

```diff
@@ -439,6 +447,44 @@ func TestPing(t *testing.T) {
 	}
 }
 
+func TestValidatePrice(t *testing.T) {
+	assert := assert.New(t)
+	mid := core.RandomManifestID()
+	b := stubBroadcaster2()
+	oinfo := &net.OrchestratorInfo{
+		PriceInfo: &net.PriceInfo{
+			PricePerUnit:  1,
+			PixelsPerUnit: 3,
+		},
+	}
+
+	s := &BroadcastSession{
+		Broadcaster:      b,
+		ManifestID:       mid,
+		OrchestratorInfo: oinfo,
+		PMSessionID:      "foo",
+	}
+
+	// B's MaxPrice is nil
+	ok := validatePrice(s)
+	assert.True(ok)
+
+	// B MaxPrice > O MaxPrice
```

Comment:
Technically, the price in `OrchestratorInfo` is O's min price and not its max price so I suggest just saying "O price" in the comments for these test cases.

Reply:
Oh yeah totally right, not intended, my bad.

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/992/commits/18648d78c0d532e7e1cc26d8bc1e374a560cee34